tests/dragonfly/cluster_test.py

```diff
@@ -907,272 +972,196 @@ async def test_random_keys():
+    assert await nodes[1].client.get("KEY5") == "value"
+    assert await nodes[1].client.get("KEY6") == "value"
+    assert await nodes[1].client.get("KEY7") == "value"
+    assert await nodes[0].client.get("KEY8") == "value"
+    assert await nodes[1].client.get("KEY9") == "value"
+    assert await nodes[1].client.get("KEY10") == "value"
+    assert await nodes[1].client.get("KEY11") == "value"
+    assert await nodes[1].client.get("KEY12") == "value"
+    assert await nodes[1].client.get("KEY13") == "value"
+    assert await nodes[1].client.get("KEY14") == "value"
+    assert await nodes[1].client.get("KEY15") == "value"
+    assert await nodes[1].client.get("KEY16") == "value"
+    assert await nodes[1].client.get("KEY17") == "value"
+    assert await nodes[1].client.get("KEY18") == "value"
+    assert await nodes[1].client.get("KEY19") == "value"
+    assert await nodes[1].client.get("KEY20") == "value"
+    assert await nodes[1].client.get("KEY21") == "value"
+    assert await nodes[1].client.execute_command("DBSIZE") == 19
+
+    await check_for_no_state_status([node.admin_client for node in nodes])
+    await close_clients(*[node.client for node in nodes], *[node.admin_client for node in nodes])
 
-    assert (
-        await c_nodes_admin[1].execute_command("DFLYCLUSTER", "SLOT-MIGRATION-STATUS") == "NO_STATE"
-    )
-    assert (
-        await c_nodes_admin[0].execute_command("DFLYCLUSTER", "SLOT-MIGRATION-STATUS") == "NO_STATE"
-    )
 
-    await close_clients(*c_nodes, *c_nodes_admin)
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_network_disconnect_during_migration(df_local_factory, df_seeder_factory):
+    instances = [
+        df_local_factory.create(port=BASE_PORT + i, admin_port=BASE_PORT + i + 1000)
+        for i in range(2)
+    ]
 
+    df_local_factory.start_all(instances)
 
-@dataclass
-class MigrationInfo:
-    ip: str
-    port: int
-    slots: list
-    node_id: str
+    nodes = [(await create_node_info(instance)) for instance in instances]
+    nodes[0].slots = [(0, 16383)]
+    nodes[1].slots = []
 
+    await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
-@dataclass
-class NodeInfo:
-    instance: DflyInstance
-    client: aioredis.Redis
-    admin_client: aioredis.Redis
-    slots: list
-    next_slots: list
-    migrations: list
-    id: str
+    seeder = df_seeder_factory.create(keys=30000, port=nodes[0].instance.port, cluster_mode=True)
 
+    await seeder.run(target_deviation=0.1)
 
-async def create_node_info(instance):
-    admin_client = instance.admin_client()
-    ninfo = NodeInfo(
-        instance=instance,
-        client=instance.client(),
-        admin_client=admin_client,
-        slots=[],
-        next_slots=[],
-        migrations=[],
-        id=await get_node_id(admin_client),
-    )
-    return ninfo
+    proxy = Proxy("127.0.0.1", 1111, "127.0.0.1", nodes[1].instance.port)
```

Comment:
Dont we block migration if  it is not on admin port?

Reply:
yes you are right I need to use admin port. Thanks

Reply:
It's not critical for this test, because we have only one migration

Reply:
I am asking if dragonfly does not block migration if it is not on admin port

Reply:
only when we have cross node migration